app/components/Toast/Toast.module.css

```diff
@@ -0,0 +1,75 @@
+@import url('~app/styles/variables.css');
+
+.toastRegion {
+  position: fixed;
+  align-items: flex-start;
+  bottom: var(--spacing-lg);
+  left: var(--spacing-lg);
+  z-index: 150;
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-sm);
+
+  @media (--mobile-device) {
+    bottom: 0;
+    left: 0;
+    right: 0;
+    gap: 0;
+  }
+}
+
+.toast {
+  display: flex;
+  align-items: center;
+  background-color: var(--color-gray-9);
```

Comment:
We could use `var(--lego-font-color)` for the background, but I thought it might be kind of bad practice to use the font color for non-font stuff. `var(--lego-card-color)` is opposite of what we want i believe (toast is dark in light mode and light in dark mode)

Reply:
I believe we use the font color for tooltips